apps/twap/src/app/routes/root.ts

```diff
@@ -0,0 +1,10 @@
+import { FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+
+export default async function (fastify: FastifyInstance) {
+  fastify.get(
+    '/',
+    async function (request: FastifyRequest, reply: FastifyReply) {
+      return { message: 'Hello API' };
+    }
```

Comment:
Aha!
The test won't fail then as we have this here.

Remove?

Reply:
Updated to a health check endpoint. It is good practice to have an endpoint that returns just a flat response, to be able to easily test if we are live with the service or not (also later on for automated tests).